### PR TITLE
The Ember Times No. 167 - November 6th 2020 

### PR DESCRIPTION
**Pull request by amyrlam**
👻 New to The Ember Times? If you want to claim a topic to write about, please comment here on the PR or chat with us in [#support-ember-times](https://discordapp.com/channels/480462759797063690/485450546887786506) on the [Ember Discord](https://discordapp.com/invite/zT3asNS)! 👻

***

### Addon/App releases

- [ ] [`ember-tracked-polyfill`](https://github.com/sukima/ember-tracked-polyfill) newly released ([source](https://discordapp.com/channels/480462759797063690/480499624663056390/741699604193607741))
- [ ] [`ember-cli-ui`](https://github.com/rajasegar/ember-cli-ui) newly released ([source 1](https://twitter.com/rajasegar_c/status/1297774711384846336), [source 2](https://twitter.com/rajasegar_c/status/1297775279553646597))
- [ ] [`ember-off-canvas-sidebar`](https://github.com/rajasegar/ember-off-canvas-sidebar) newly released ([source](https://twitter.com/rajasegar_c/status/1301116927247441920))
- [ ] EmberQuest, a geocaching online RPG written in Ember ([source](https://twitter.com/danmonroe/status/1296910382305599488))
  - https://emberquest.danmonroe.dev/
  - https://www.youtube.com/watch?v=Ld1xnQWkqPU
- [ ] [`ember-apollo-client`](https://github.com/ember-graphql/ember-apollo-client/releases/tag/v3.0.0) v3 released ([source](https://twitter.com/josemarluedke/status/1309571840860327936))
- [ ] ~~ember-bootstrap 4.2.0 is out, entering the final phase of our Octane transition! The first bunch of components have been refactored to Glimmer components. The interesting part here is that this was possible without any breaking changes!* :scream: #emberjs https://twitter.com/simonihmig/status/1310674791410675712~~ Ember Bootstrap v4.4.0 was released yesterday. Continuing our incremental Octane rewrite. Details could be found in changelog: https://github.com/kaliber5/ember-bootstrap/blob/master/CHANGELOG.md
- [ ] ember-page title and related rfc, see https://twitter.com/melaniersumner/status/1313332347698376704 / https://twitter.com/emberjs/status/1314639063861821442
- [x] From @ BryanCrotaz on Discord: I've just submitted a big PR which brought ember-key-manager up to v1.0.0 and now fully supports Octane, including template modifiers that can hook key presses to run component code. it's now been released as 1.0.0, so you could change it to a release announcement. https://www.npmjs.com/package/ember-key-manager
- [x] from @ Gaiety on Discord: ember-select-light hit 2.0.0 with octane support along with it's connection to ember-component-patterns (working on right now). Could be some big a11y wins for ember app developers who default to other less accessible select addons (🔒 @hergaiety)
- [x] from @locks on discord: https://twitter.com/emberjs/status/1322271854875365377 Today, we've decided to move RFC 674 "Deprecate transition methods of Controller and Route" into Final Comment Period. Get your comments in now! (check if it's going merged next friday though) (🔒 @ijlee2)
- [x] [4.0.0 Release of ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing/releases/tag/v4.0.0) (🔒 @chrisrng )

### Articles

- [ ] [How Does Ember's Dependency Injection System Work?](https://nullvoxpopuli.com/2020-08-08-how-does-di-work/)
- [ ] [Simplify Your Ember App with Domain Components](https://mattfrankjames.com/blog/ember-domain-components/)
- [ ] https://ilyaradchenko.com/ember-context-and-shared-parent-state/
- [ ] Ember fancy new reactive system from nullvoxpopuli https://twitter.com/nullvoxpopuli/status/1308488990371319809
- [ ] [Ember Data Live Updates with WebSockets](https://codingitwrong.com/2020/10/02/ember-data-live-updates-with-rails.html)
- [ ] [Making a modal dialog using yieldable named blocks](https://www.balinterdi.com/blog/making-a-modal-dialog-using-yieldable-named-blocks/)

### Readers' Questions

tbd

### Miscellaneous

tbd

## 📆  Future stories

- [ ] I created one large issue with almost all ember-related articles across last 12 months 
 https://discord.com/channels/480462759797063690/480777444203429888/770366277083463702
- https://twitter.com/JohnRodkey/status/1300608539912794113
  > John Rodkey on Twitter: We will be blogging about the move and putting on something other than Medium so there isn't a paywall.  The main reasons, the community, batteries included, and strong conventions.

## 🤔  Questions?

See #support-ember-times on Discord or CONTRIBUTING.

## ✅  Publishing TODOs:

- [ ] Ensure no use of Markdown italic, use `<span style="font-style: italic;">` instead 
- [ ] Check Netlify preview e.g. buttons, to ensure display on emberjs.com/blog is good
- [ ] Crosspost on dev.to, delete any alex, redo any span italics, manually redo Readers Question picture, add Canonical URL 
- [ ] Set up tweet for Monday morning on tweetdeck
- [ ] Post $natural for #news-and-announcements in Discord

### CODE REVIEW

**Comment by jelhan** [comment]
>     * ember-bootstrap 4.2.0 is out, entering the final phase of our Octane transition! The first bunch of components have been refactored to Glimmer components. The interesting part here is that this was possible without any breaking changes!* 😱 #emberjs https://twitter.com/simonihmig/status/1310674791410675712

This is a little bit outdated. Ember Bootstrap v4.4.0 was released yesterday. Continuing our incremental Octane rewrite. Details could be found in changelog: https://github.com/kaliber5/ember-bootstrap/blob/master/CHANGELOG.md

**Comment by amyrlam** [comment]
@jelhan thank you very much! updated above and we'll try to write about that for friday's issue. (you're always welcome to submit a writeup as well, if you like)

**Comment by hergaiety** [comment]
I am happy to claim writing about my own addon's updated: `ember-select-light hit 2.0.0`

**Comment by chrisrng** [comment]
> I am happy to claim writing about my own addon's updated: `ember-select-light hit 2.0.0`

Go for it thanks @hergaiety !

File: source/2021-11-06-the-ember-times-issue-167.md [hunk 1]
```diff
@@ -1,27 +1,26 @@
 ---
 title: The Ember Times - Issue No. 167
-author: the crowd
+author: Chris Ng, the crowd
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2020
-alias : "blog/2021/11/06-the-ember-times-issue-167.html"
+alias : "blog/2020/11/06-the-ember-times-issue-167.html"
```
Comment (hergaiety) [inline source/2021-11-06-the-ember-times-issue-167.md]: This `alias` is correct, but the filename for this file is wrong. The year is `2021` but it should be `2020`.

**Review by hergaiety — CHANGES_REQUESTED** [review]

File: source/2021-11-06-the-ember-times-issue-167.md (hunk 1 above)
Comment (chrisrng) [reply source/2021-11-06-the-ember-times-issue-167.md]: good catch fixed that!

**Pull request merged**